conf/packages

```diff
@@ -1,14 +1,13 @@
-libgdal1 | libgdal20
+libgdal1 | libgdal20 | libgdal28
```

Comment:
Do you want to change this to libgdal-dev and drop the python3- packages as we're doing in https://github.com/mysociety/mapit/pull/388? Only query now I think about that is we dropped python3-memcache and python3-gdal from that PR, but it looks like the code would still use them if present, so not sure if that was a mistake.